### ISSUE TITLE
Mention style doesn't work correctly

### ISSUE THREAD

**Issue by TaiTrien** [dabakovich/react-native-controlled-mentions#66]
When mentioned text has an icon at the end of it, the style of normal text will follow to mention style. So any ideas? The screen shoot is attached below
![Image from iOS](https://user-images.githubusercontent.com/32502320/122030428-f79fe700-cdf7-11eb-9b8b-7f7b97dcd2cb.png)

**Comment by griarman** [comment]
any updates?

**Comment by dabakovich** [comment]
Thanks for your feedbacks. I'm working on the next major update and will try to include a fix for this bug.

**Comment by tiranavayan** [comment]
Thanks 👍🙏

**Comment by dabakovich** [comment]
@tiranavayan alpha v3 is released now. Can you test is the bug still reproducing on new version?

Please, use this [branch](https://github.com/dabakovich/react-native-controlled-mentions/tree/3.0.0-feat-use-mentions) to get usage info and example.

You can install alpha version using those commands:
```
yarn add react-native-controlled-mentions@alpha
// or
npm install --save react-native-controlled-mentions@alpha
```

**Comment by dabakovich** [comment]
@tiranavayan can you install updated `alpha` version and test how the input work with emojis now?

```
yarn add react-native-controlled-mentions@alpha
// or
npm install --save react-native-controlled-mentions@alpha
```

**Comment by dabakovich** [comment]
I'm getting the same issue. It's look like it is an iOS bug. As workaround, you can use option `isInsertSpaceAfterMention`. With space after mention it should work as expected.

**Comment by tiranavayan** [comment]
I use this prop, but after deleting space I'm getting same issue ((

**Comment by dabakovich** [comment]
Yeah, that sad. Unfortunately, I can't do with this bug anymore yet.

**Comment by tiranavayan** [comment]
So sad

**Comment by arpankumarlahiri** [comment]
any update? am facing same issue

**Comment by fukemy** [comment]
any update @dabakovich , you can test it by typing fast

**Comment by nicolasforeros** [comment]
I think this is an issue related to react-native: [https://github.com/facebook/react-native/issues/29890](issue)

**Comment by habrahamyanmabbly** [comment]
Facing the same problem, has anyone been able to fix it?

**Comment by dabakovich** [comment]
The bug is fixed in [alpha2](https://github.com/dabakovich/react-native-controlled-mentions/releases/tag/v3.0.0-alpha.2), and not reproducible anymore.

Closing the issue, feel free to add comments if something will not work for you.
